modules/sovrnBidAdapter.js

```diff
@@ -234,15 +234,15 @@ export const spec = {
             dealId: sovrnBid.dealid || null,
             currency: 'USD',
             netRevenue: true,
-            mediaType: sovrnBid.nurl ? BANNER : VIDEO,
+            mediaType: sovrnBid.mtype == 2 ? VIDEO : BANNER,
             ttl: sovrnBid.ext?.ttl || 90,
             meta: { advertiserDomains: sovrnBid && sovrnBid.adomain ? sovrnBid.adomain : [] }
           }
 
-          if (sovrnBid.nurl) {
-            bid.ad = decodeURIComponent(`${sovrnBid.adm}<img src="${sovrnBid.nurl}">`)
-          } else {
+          if (sovrnBid.mtype == 2) {
             bid.vastXml = decodeURIComponent(sovrnBid.adm)
+          } else {
+            bid.ad = decodeURIComponent(`${sovrnBid.adm}<img src="${sovrnBid.nurl}">`)
```

Comment:
Can we move this functionality into Blackbird as well?  Could we have a line that's just:

`bid.ad = decodeURIComponent(sovrnBid.adm)`

Reply:
Yeah, after thinking about it more I think this comment doesn't actually work due to the legacy prebid versions - I agree.  I'll close this comment.